src/main/java/com/daml/extensions/testing/ledger/DamlScriptRunner.java

```diff
@@ -90,10 +92,12 @@ public DamlScriptRunner build() {
 
     private ProcessBuilder command() {
       String sandboxHost = "localhost";
+      String damlCommand = OS.isWindows()? "daml.cmd" : "daml";
```

Comment:
_Minor_: I think it would make sense to encapsulate this into `OS`. Something like:

```java
public static String damlCommand() {
  return isWindows() ? "daml.cmd" : "daml";
}
```

Reply:
yes, I think that is better, updated